### PR TITLE
fix: antd v4 upgrade

### PR DESCRIPTION
**Pull request by SavvyShah**
[Notion card](https://www.notion.so/reactivesearch/Develop-Browse-Data-91be3cdbf44d4c97ab50c3390f20dd36)
- Upgrade the usage of `antd` from v3 to v4

[Batteries PR](https://github.com/appbaseio/batteries/pull/312)

### CODE REVIEW

File: packages/browser/src/index.js [hunk 1]
```diff
@@ -24,14 +24,12 @@ import * as constants from './constants';
...
 import colors from './components/theme/colors';
 
-import './antd.css';
-
 // shared store
 const store = configureStore();
 
 function WithConfigProvider(props) {
 	return (
-		<ConfigProvider prefixCls="dejavu-browser">
```
Comment (mohdashraf010897) [inline packages/browser/src/index.js:34]: @SavvyShah Why did you remove `prefixCls` prop?

**Review by mohdashraf010897 — CHANGES_REQUESTED** [review]

File: packages/browser/src/index.js (hunk 1 above)
Comment (SavvyShah) [reply packages/browser/src/index.js:34]: This is why we were having a [separate stylesheet before](https://github.com/appbaseio/dejavu/pull/451/files#diff-6b5ccd864a497944e16cd0dab1a78d4c936b974d4bca3c1168830cd700f18bb7). This doesn't allow the user of package to style the components.
